Modules/_ctypes/_ctypes_test.c

```diff
@@ -13,6 +13,10 @@
 
 #include <Python.h>
 
+#ifdef HAVE_C_COMPLEX
+#  include "../_complex.h"        // csqrt()
+#  undef I                        // for _ctypes_test_generated.c.h
```

Comment:
```suggestion
#  undef I                        // _Complex_I
```

Reply:
Here I would like to emphasize where new macro create a problem.

Reply:
Can we avoid the relative include? They tend to produce problems further down the road.

Reply:
How about ``Include/internal/pycore_pycomplex.h``?

There is ``Include/internal/pycore_pymath.h``, that (IMHO) does part of work ``Modules/_math.h`` (``<math.h>`` workarounds).  There are no relative includes of ``_math.h`` (yet), but I think these files could be merged.

Reply:
I feel that `_math.h` and `_complex.h` could be named as `Include/internal/pylibc_math.h` because what they do is patching the C standard library itself rather than adding functionalities.

Reply:
@vstinner: I'd really like to avoid the relative include, but we can fix that in a follow-up PR.